app/src/org/odk/collect/android/utilities/NepaliDateHelper.java

```diff
@@ -0,0 +1,24 @@
+package org.odk.collect.android.utilities;
+
+import java.util.Date;
+
+import org.commcare.dalvik.R;
+
+import android.content.Context;
+
+/**
+ * Helper to convert a Gregorian Date to a Nepali date, formatted as a 'd MMMM yyyy' string.
+ * 
+ * @author Richard Lu
+ */
+public class NepaliDateHelper {
```

Comment:
I am not sure; this class exists in the package where EthiopianDateHelper exists, which is used by CalendaredDateFormatHandler. Perhaps EthiopianDateHelper exists separately only because EthiopianDateWidget was using JodaTime's existing EthiopianChronology implementation (rather than our own implementation), in which case there is no good reason for NepaliDateHelper to exist separately from NepaliDateUtilities. Will combine